js/apps/admin-ui/test/realm-settings/oid4vci-attributes.spec.ts

```diff
@@ -48,8 +54,11 @@ test("OID4VCI tab visibility", async ({ page }) => {
 test("should render fields and save values with correct attribute keys", async ({
   page,
 }) => {
-  // Ensure OID4VCI tab is available and click it
-  await ensureOid4vciTabAvailable(page);
+  const oid4vciTab = await getOid4vciTab(page);
+  if (!oid4vciTab) return;
```

Comment:
I understand the situation and agree with the move to directly rely on whether the feature is enabled or not. However, you could maybe revert to  `test.skip()` instead of `return` so it is clear in test reports that the test was skipped.

```suggestion
  const oid4vciTab = await getOid4vciTab(page);
  if (!oid4vciTab) test.skip();
```